docs/development/release_management.rst

```diff
@@ -83,10 +78,12 @@ Pre-Release
 
 #. For each release candidate, update the version and changelog.
 
-   a. Collect a list of important changes from the `SecureDrop milestones
-      <https://github.com/freedomofpress/securedrop/milestones>`_ for the release, including
-      GitHub issues or PR numbers for each change. You will add these changes to the changelog in
-      the next step.
+   a. Collect a list of changes since the last release. For example, if the last release was version
+      1.6.0, you can view changes in Github by running:
+      `https://github.com/freedomofpress/securedrop/compare/release/1.6.0...develop`. Also check
```

Comment:
This is currently italicized, if you meant to display it as preformatted text, use double backtics (I think preformatting may make it clearer that the `.` is not part of the URL).